### ISSUE TITLE
 "Train in Azure" option cannot be focused with the "Tab" key.

### ISSUE THREAD

**Issue by vzhuqin** [dotnet/machinelearning-modelbuilder#1575]
**System Information (please complete the following information):**
- Windows OS: Windows-10-Enterprise-21H1
- ML.Net Model Builder (Preview): 16.6.2.2135501 (Main Branch)
- Microsoft Visual Studio Enterprise 2019: 16.10.2
- .Net: 5.0.301

**Describe the bug**
- On which step of the process did you run into an issue: "Next steps" page
- Clear description of the problem: Clicking "Tab" key,  "Train in Azure" cannot be focused.

**To Reproduce**

1. Select Create a new project from the Visual Studio 2019 start window;
2. Choose the C# Console App (.NET Core) project template with .Net 5.0;
3. Add model builder by right click on the project;
4. Select "Image classification" or "Object detection" scenario which support Azure environment to train the model;
5. Pass all and navigate to "Next steps" page;
6. Clicking "Tab" key, find cannot focus on "Train in Azure" option.

**Expected behavior**
All options can be focused with the "Tab" key.

**Screenshots**
![cannot focus Train in Azure](https://user-images.githubusercontent.com/81727020/125047850-f6e63200-e0d1-11eb-9ab3-c24c63ac39af.gif)

**Additional context**

**Comment by vzhuqin** [comment]
Not repro this issue on latest main build: 16.7.2.2140901
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/92e426d7-b836-4af2-adc4-178076027de4)
